teamA/leeHyeonTae/db/migrations/20221007141640_posts.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE posts(
+    id INT NOT NULL AUTO_INCREMENT,
+    title VARCHAR(100) NOT NULL,
+    content VARCHAR(1000) NOT NULL,
+    posting_image VARCHAR(2000) NULL,
```

Comment:
해당 테이블의 필드를 조회할 때 다음과 같습니다.
`posts.posting_image`

posts와 관련된 키워드가 중복되고 있습니다.

다음과 같이 수정하여 중복을 줄여주세요~!
```suggestion
    image_url VARCHAR(2000) NULL,
```